### ISSUE TITLE
autodownload only executes once

### ISSUE THREAD

**Issue by cccs-cat001** [nbgallery/lab-extensions#11]
Hi. I'm playing with these extensions on our nbgallery/jupyterhub instances. When I first launch my jupyterhub server with this autodownload installed, and go to the tree view, it will download all the starred notebooks (and probably recently executed but I don't have any so there's none in there). 

So I starred more notebooks on nbgallery, but I can't seem to make jupyterhub download those.. I've tried restarting my server, refreshing, restarting the autodownload extension. None seem to make it work. Any idea why this happens?

**Comment by jeffyjefflabs** [comment]
Yeah, the original intent was to help recreate your Jupyter environment when you're on ephemeral VMs that get recreated from scratch periodically.  So the idea was just to run once at startup to fetch the notebooks you use frequently.

**Comment by mcrutch** [comment]
Was looking back at tickets and I have a possible idea on this, but it gets complicated.  
Possible option: Periodic (obviously not every time, 1/day probably) check to download any NEW notebooks but won't touch existing.  Can squirrel this away in settings to track when the last check was.  Open to thoughts.  Since it would only happens when the whole UI loads, it can be non-invasive.
